config/index.js

```diff
@@ -1,5 +1,18 @@
-const envs = ['development', 'production', 'test']
+export const host = process.env.HOST;
+export const port = process.env.PORT;
+export const env = process.env.NODE_ENV;
 
-export const host = process.env.HOST || 'localhost';
-export const port = process.env.PORT || '3000';
-export const env = envs[0];
+export const session = {
+    collection: 'sessions',
+    name: process.env.APP_NAME,
+    secret: process.env.SESSION_SECRET,
+    cookie: {
+        path: '/',
+        httpOnly: true,
+        secure: false,
+        maxAge: 86400000    // 1 hour
+    },
+    resave: false,  //don't save session if unmodified
+    saveUninitialized: true, // don't create session until something stored
+    rolling: true   //Force a cookie to be set on every response. This resets the expiration date.
+}
```

Comment:
I meant move all config to app index.js. we use dotenv to store env variables so no need to have this config file anymore